Workbooks/WorkbooksMetadata.json

```diff
@@ -674,5 +674,57 @@
     "templateRelativePath": "Perimeter81OverviewWorkbook.json",
     "subtitle": "",
     "provider": "Perimeter 81"
+  },
+  {
+    "workbookKey": "SymantecProxySGWorkbook",
+    "logoFileName": "",
+    "description": "Gain insight into Symantec ProxySG by analyzing, collecting and correlating proxy data.\nThis workbook provides visibility into ProxySG Access logs",
+    "dataTypesDependencies": ["SymantecProxySG"],
+    "dataConnectorsDependencies": [ "Symantec ProxySG" ],
+    "previewImagesFileNames": [ "SymantecProxySGWhite.png", "SymantecProxySGBlack.png" ],
+    "version": "1.0",
+    "title": "Symantec ProxySG",
+    "templateRelativePath": "SymantecProxySG.json",
+    "subtitle": "",
+    "provider": "Symantec"
+  },
+  {
+    "workbookKey": "PulseConnectSecureWorkbook",
+    "logoFileName": "",
```

Comment:
Pulse Connect Secure doesn't have a logo file

Reply:
unable to find a logo for product